packages/kit/src/views/Receive/pages/ReceiveToken.tsx

```
@@ -47,7 +47,7 @@ import { EConfirmOnDeviceType } from '@onekeyhq/shared/types/device';
 
 import backgroundApiProxy from '../../../background/instance/backgroundApiProxy';
 import AddressTypeSelector from '../../../components/AddressTypeSelector/AddressTypeSelector';
-import { HyperlinkText } from '../../../components/HyperlinkText';
+import { FormatHyperlinkText, HyperlinkText } from '../../../components/HyperlinkText';
```

ESLint

[error] Replace `·FormatHyperlinkText,·HyperlinkText·` with `⏎··FormatHyperlinkText,⏎··HyperlinkText,⏎`

(prettier/prettier)